lib/explorer/series.ex

```diff
@@ -1833,17 +1840,24 @@ defmodule Explorer.Series do
     apply(impl, fun, [series | args])
   end
 
-  defp check_types(list) do
-    {last_item, type, types_match?} =
-      Enum.reduce_while(list, {nil, nil, true}, &check_types_reducer/2)
+  def check_types(list) do
```

Comment:
Do you want to `@doc false` those functions maybe? They are not public API, right? Or perhaps move to a module you can share them?

Reply:
Good point!

Reply:
How's that now?